src/pages/product/product.tsx

```diff
@@ -0,0 +1,99 @@
+import Header from '../../components/header/header';
+import CardRate from '../../components/card-rate';
+import ProductTabs from '../../components/product-tabs';
+import ReviewBlock from '../../components/review-block';
+import ButtonUp from '../../components/button-up';
+import Footer from '../../components/footer';
+import ProductSimilar from '../../components/product-similar/product-similar';
+
+import { useParams, Link } from 'react-router-dom';
+import { useEffect } from 'react';
+import { useAppSelector} from '../../hooks';
+import { fetchCameraAction, fetchRequests } from '../../store/api-actions';
+import { store } from '../../store';
+
+function Product(): JSX.Element {
+  const params = useParams();
+  const cameraId = params.id;
+
+  useEffect(() => {
+    store.dispatch(fetchCameraAction(cameraId));
+    store.dispatch(fetchRequests(cameraId));
+  }, [cameraId]);
+
+  const camera = useAppSelector((state)=>state.camera);
+  const reviews = useAppSelector((state)=>state.reviews);
+
+  if(!camera) {
+    return <> Loading</>;
+  }
+  if(!reviews) {
+    return <> Loading</>;
+  }
```

Comment:
```suggestion
  if(!camera || !reviews) {
    return <> Loading</>;
  }
```
А вообще по хорошему показывать 2 отдельных лоадера на  обзор и на камеры.